File: .NET Core/Exceptions/InvalidCredentialsBackendException.cs
using com.contidio.sdk.proto;
using System.Net;

namespace Contidio.Sdk.Exceptions
{
    public class InvalidCredentialsBackendException : BackendException
    {
        public InvalidCredentialsBackendException()
            : base(BackendErrorCode.INVALID_CREDENTIALS, "Invalid credentials", "InvalidCredentialsBackendException", HttpStatusCode.Unauthorized)
        {
        }
    }
}

File: Java/src/main/java/com/contidio/sdk/exceptions/InvalidCredentialsBackendException.java
package com.contidio.sdk.exceptions;

import org.apache.commons.httpclient.HttpStatus;

import com.contidio.sdk.proto.MetadataProtocol.BackendErrorCode;

@SuppressWarnings("serial")
public class InvalidCredentialsBackendException extends BackendException {
	public InvalidCredentialsBackendException() {
		super(BackendErrorCode.INVALID_CREDENTIALS, "Invalid credentials", "InvalidCredentialsBackendException", 
				HttpStatus.SC_UNAUTHORIZED);
	}
}
